types/m3u8-parser/package.json

```diff
@@ -9,6 +9,10 @@
         "@types/m3u8-parser": "workspace:."
     },
     "owners": [
+        {
+            "name": "mkzhx",
+            "githubUsername": "mkzhx"
+        },
```

Comment:
How come this user is being added as an owner? They don't seem to be active:

![Screenshot of mkzhx's GitHub profile](https://github.com/user-attachments/assets/0f7633e6-6ead-48dc-8fdc-575b9d635f87)

Reply:
Please don’t add owners other than yourself without their consent or some kind of explanation.